src/test/java/org/folio/rest/impl/GOBIIntegrationServiceResourceImplTest.java

```diff
@@ -491,11 +492,82 @@ public final void testPostContentWithValidOkapiToken(TestContext context) throws
     logger.info("End: Testing for 201 - posted order listed print monograph");
   }
 
+  @Test
+  public final void testPostGobiOrdersFallBackMaterialTypes(TestContext context) throws Exception {
+    logger.info("Begin: Testing for falling back to the unspecified material id, if a non existent code is sent");
+
+    final Async asyncLocal = context.async();
+
+    final String body = getMockData(POLISTEDELECTRONICSERIALPATH);
+
+    final GobiResponse order = RestAssured
+      .given()
```

Comment:
Here is where the code formatter fails us...  It's admittedly a nitpick, but we should try to avoid the flat indentation of these restAssured blocks... it's harder to read this way.